nvme.spec.in

```diff
@@ -14,19 +13,13 @@ BuildRoot:	%{_tmppath}/%{name}-%{version}-root
 NVMe is a fast, scalable, direct attached storage interface. The nvme
 cli rpm installs core management tools with minimal dependencies.
 
-%prep
-%setup
-
-%build
-make
-
 %install
-rm -rf $RPM_BUILD_ROOT
-make install-spec DESTDIR=%{buildroot} PREFIX=/usr
+meson install --destdir %{buildroot} --skip-subprojects
+touch %{buildroot}/%{_sysconfdir}/nvme/hostnqn
+touch %{buildroot}/%{_sysconfdir}/nvme/hostid
```

Comment:
Is this a good idea to add empty hostnqn and hostid files? They should be generated on installation time, IMO.

Reply:
This needs to be listed up in files of rpm installs.
In package should be exist.
And on %post it will be generated as you mentioned :)